apps/admin-ui/src/component/Navigation.tsx

```diff
@@ -1,131 +1,229 @@
+            style={{ filter: "invert(1)" }}
+          />
+        }
+        logoAriaLabel="Varaamo logo"
+        logoHref="/"
```

Comment:
Does this work? not require `/kasittely`? i.e. `env.NEXT_PUBLIC_BASE_URL`

Reply:
fixed